sippy/sippy/ingress.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
I'm not sure if aaa is at a version that supports this

Reply:
good catch - the cluster is still in 1.16 (will be upgraded to 1.17 soon, but ingress/v1 is only in 1.19), so we should use v1beta1 stil...